package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@pocket-tools/backend-benchmarking",
+  "version": "1.0.0",
```

Comment:
I could be wrong but I think the version has to be left as `0.0.0-development` for semantic release